CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ x.y.z Release notes (yyyy-MM-dd)
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-swift/issues/????), since v?.?.?)
-* None.
+* Fixed a crash that would occur when an http error 401 or 403 is returned upon opening a watch stream for a MongoDB collection. ([#8519](https://github.com/realm/realm-swift/issues/8519))
```

Comment:
This probably needs a line jump

Reply:
Sure - I typically don't add line breaks in markdown files, but can see this is how the changelog is written already, so might as well stick to the precedent in the swift repo.